content/recipes/introduction/FAIR-and-knowledge-graphs.md

```diff
@@ -100,51 +133,17 @@ Data Use
 RDF Graph and Labeled Property graphs
 ````
 
-
-
-```{admonition} **So how does graph theory come to meet knowledge representation?**
-
-We need here to introduce the definition of a graph:
-
-A graph is a set of nodes (also known as vertex) connected via edges. 
-When the edges connected any two nodes carry a direction information, they defined a specific and important subfamily
-of graphs known as `directed graphs`.
-When cycles aren't allowed, a new subtype of graph is defined. Such graphs are known as `directed acyclic graphs`.
-
-Volumes have been written about what graphs are and the underlying mathematical theories describing their properties. 
-The readership can refer to the following resources for a complete overview and training. 
-The purpose of this section is simply to provide a rapid overview of what knowledge graphs are and their typology,
-as one needs to be aware of the differences that exist between the different kinds of graphs and also understand the 
-consequences both theoretical and technical to fully appreciate the pros and cons of the various implementations
-available out there.
-The following sections intend to provide a compact review and guide newcomers through the jargon and the various
-acronyms founds in the literature on the topic.
-```
-
 ### 1. RDF graphs
 
-
 #### i. Defining RDF, Resource Description Framework
-
 Here, we need to introduce the notion of representing statements as a predicate of the following form:
-```bash
-<subject><predicate><object>
-```
-
-such as:
-
-```bash
-`P53 protein` `interacts with` `DNA`.
-```
+`<subject><predicate><object>`, such as: `'P53 protein' 'interacts_with' 'DNA'`.
 
 This statement can be formally expressed using a dedicated syntax called [RDF](https://www.w3.org/TR/rdf11-concepts/), 
-standing for "resource description framework", one of the W3C standards supporting the vision of a **Semantic Web** as 
+standing for "Resource Description Framework", one of the W3C standards supporting the vision of a **Semantic Web** as 
 outlined by Sir Tim Berners-Lee and colleagues {footcite}`bernerslee2001semantic`.
```

Comment:
I expected the footcite hyperlink would bring me to the corresponding section in the reference. but nothing happened when I clicked on the footnote